File: Stack/Java/src/app/Stack.java
package app;

import java.util.Arrays;

public class Stack {

    private int[] items =  new int[5];
    private int count;

    public void push(int item){
        if(count == items.length)
            throw new StackOverflowError();
        items[count++] = item;
    }

    public int pop(){
        if(count == 0)
            throw new IllegalArgumentException();

        return items[--count];
    }

    public int peek(){
        if(count == 0)
            throw new IllegalArgumentException();
        return items[count - 1];
    }

    public Boolean isEmpty(){
        return count == 0;
    }

    @Override
    public String toString(){
        var content = Arrays.copyOfRange(items, 0, count);
        return Arrays.toString(content);
    }

}

File: Stack/Python/Stack.py

class Stack(object):
    def __init__(self):
        self.data = []
        self.count = 0

    def push(self, item):
        self.data.append(item)
        self.count += 1

    def pop(self):
        if self.count == 0:
            return "Stack is Empty"

        data = self.data[-1]
        del self.data[-1]
        self.count -= 1
        return data

    def peek(self):
        if self.count == 0:
            return "Stack is Empty"
        return self.data[-1]

    def isEmpty(self):
        return self.count == 0

    def size(self):
        return self.count
